.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - dev
```

Necessary to test on main and dev, though we could add the individual branches (maybe not necessary if testing new feat tho)
```suggestion
    branches:
      - main
      - dev
      - ethan/*
      - callum/*
      - rafay/*
      - adham/*
```